lms/templates/ccx/enrollment.html

```diff
@@ -4,16 +4,16 @@
 from openedx.core.djangolib.markup import HTML, Text
 %>
 
-<h2 class="hd hd-2">${_("Batch Enrollment")}</h2>
-<div class="batch-enrollment" style="float:left;width:100%">
+<h2 class="hd hd-2">${_("Enrollment")}</h2>
+<div class="batch-enrollment-ccx">
   <form method="POST" action="ccx-manage-students">
   <input type="hidden" name="csrfmiddlewaretoken" value="${ csrf_token }">
   <label for="student-ids" class="sr">${_("Email Addresses/Usernames")}</label>
   <p id="label_student_ids" class="text-helper">
-      ${_("Enter email addresses and/or usernames separated by new lines or commas.")}
+      ${_("Enter one or more email addresses and/or usernames separated by new lines or commas.")}
```

Comment:
@srpearce -- how does this copy look to you?

Reply:
I'd make a few changes (I don't think a docs person looked at any of this UI text before it was added to the platform!):

Enter one or more email addresses or usernames separated by new lines or commas.

Reply:
Would it also be possible for line 14 to be changed? Suggestion:

Make sure you enter the information carefully. You will not receive notification for invalid usernames or email addresses.